examples/build-package/main.tf

```diff
@@ -17,17 +17,50 @@ resource "random_pet" "this" {
 # Build packages
 #################
 
-# Create zip-archive of a single directory where "pip install" will also be executed (default for python runtime)
+# Create zip-archive of a single directory where "pip install" will also be executed (default for python runtime with requirements.txt present)
 module "package_dir" {
   source = "../../"
 
   create_function = false
 
-  runtime     = "python3.8"
-  source_path = "${path.module}/../fixtures/python3.8-app1"
+  build_in_docker = true
+  runtime         = "python3.8"
+  source_path     = "${path.module}/../fixtures/python3.8-app1"
+  artifacts_dir   = "${path.root}/builds/package_dir/"
+}
+
+# Create zip-archive of a single directory where "poetry install" will also be executed
+module "package_dir_poetry" {
+  source = "../../"
+
+  create_function = false
+
+  build_in_docker = true
+  runtime         = "python3.8"
+  source_path = [
+    {
+      path = "${path.module}/../fixtures/python3.8-app-poetry"
+      patterns = [
+        "!(.*/)*__pycache__/.*",
+        "!_distutils_hack/.*",
+        "!.pytest_cache/.*",
+        "!.venv/.*",
+        "!_virtualenv.pth",
+        "!_virtualenv.py",
+        "!distutils-precedence.pth",
+        "!easy_install.py",
+        "!(pip|pip-.*)(\\.virtualenv|/.*)",
+        "!(pkg_resources|pkg_resources-.*)(\\.virtualenv|/.*)",
+        "!(setuptools|setuptools-.*)(\\.virtualenv|/.*)",
+        "!(wheel|wheel-.*)(\\.virtualenv|/.*)",
```

Comment:
This is required as poetry does not support a `--target` parameter like `pip` so everything is put in the same directory of the created virtualenv.